pkgs/development/tools/symfony-cli/default.nix

```diff
@@ -25,8 +27,12 @@ buildGoModule rec {
     "-X main.channel=stable"
   ];
 
+  buildInputs = [ makeBinaryWrapper ];
+
   postInstall = ''
     mv $out/bin/symfony-cli $out/bin/symfony
+
+    wrapProgram $out/bin/symfony --prefix PATH : ${lib.makeBinPath [ nssTools ]}
```

Comment:
Maybe something like this (I haven't tested):

```shell
makeWrapper $out/bin/symfony-cli $out/bin/symfony \
  --prefix PATH : ${lib.makeBinPath [ nssTools ]}
```

Reply:
It doesn’t work. What do you think about this ?

```
  postInstall = ''
    mkdir $out/libexec
    mv $out/bin/symfony-cli $out/libexec/symfony

    makeBinaryWrapper $out/libexec/symfony $out/bin/symfony \
      --prefix PATH : ${lib.makeBinPath [ nssTools ]}
  '';
```

Reply:
LGTM

Reply:
Ok. So let’s go !